src/metabase/driver/mysql.clj

```diff
@@ -294,6 +294,16 @@
        (-> (dbspec/mysql details)
            (sql-jdbc.common/handle-additional-options details))))))
 
+(defmethod sql-jdbc.sync/accessible-tables-for-user :mysql
+  [_ db-or-id-or-spec user]
+  (jdbc/query (sql-jdbc.conn/db->pooled-connection-spec db-or-id-or-spec)
+              [(format "SELECT table_name, table_schema AS table_schem
+                        FROM information_schema.table_privileges
+                        WHERE grantee like \"'%s'@%%\"
```

Comment:
can this be done with a bound parameter? doing formats in sql statements always makes me nervous.

Something like:

`WHERE grantee LIKE ?` and `(format "\"'%s'@%%\" user)` - basically shift the format.